tools/tertiary-analysis/dropletutils/dropletutils-read-10x.xml

```diff
@@ -13,12 +13,32 @@ dropletutils-read-10x-counts.R
     -s .
     -c TRUE
     -o '${output_rds}'
+
+#if $metadata_files 
+--metadata-files "${metadata_files}" 
+#end if
+#if $cell_id_column 
```

Comment:
is cell_id_column optional if the boolean add_metadata_file was set to true? If that is the case, set the param to optional. Otherwise, I would use a single if for the two fields, probably based on the boolean add_metadata_file that you wrote.